codegenerator/cli/templates/dynamic/codegen/src/Types.res.hbs

```diff
@@ -230,72 +267,110 @@ module HandlerTypes = {
   type handler<'eventArgs, 'loaderReturn> = handlerArgs<'eventArgs, 'loaderReturn> => promise<unit>
 
   @genType
-  type loaderHandler<'eventArgs, 'loaderReturn> = {
+  type loaderHandler<'eventArgs, 'loaderReturn, 'eventFilter> = {
     loader: loader<'eventArgs, 'loaderReturn>,
     handler: handler<'eventArgs, 'loaderReturn>,
+    wildcard?: bool,
+    eventFilters?: SingleOrMultiple.t<'eventFilter>,
   }
```

Comment:
I wasn't able to use the optional second argument trick for gentype for the loaderWithHandler registration function. The problem was because 'loaderReturn remains a polymorphic parameter and using genType.import for the function signature wouldn't allow me to have a scoped polymorphic type. So basically I thought it's better to just extend the options into this structure. 

Let me know if that makes sense.